python-template/{{cookiecutter.placeholder_repo_name}}/python_library_tree.txt

```diff
@@ -0,0 +1,53 @@
+.
+├── .DS_Store
+├── .coveragerc
+├── .editorconfig
+├── .github
+│   ├── ISSUE_TEMPLATE.md
+│   └── PULL_REQUEST_TEMPLATE.md
+├── .gitignore
+├── .travis.yml
+├── CHANGELOG.rst
+├── LICENSE.txt
+├── MANIFEST.in
+├── Makefile
+├── README.rst
+├── docs
+│   ├── Makefile
+│   ├── __init__.py
+│   ├── _static
+│   │   └── theme_overrides.css
+│   ├── changelog.rst
+│   ├── conf.py
+│   ├── getting_started.rst
+│   ├── index.rst
+│   ├── internationalization.rst
+│   ├── make.bat
+│   ├── readme.rst
+│   └── testing.rst
+├── openedx.yaml
+├── pylintrc
+├── pylintrc_tweaks
+├── python_library_tree.txt
+├── requirements
+│   ├── .DS_Store
+│   ├── base.in
+│   ├── constraints.txt
+│   ├── dev.in
+│   ├── doc.in
+│   ├── monitoring
+│   │   └── requirements.txt
+│   ├── optional.txt
+│   ├── pip-tools.in
+│   ├── pip-tools.txt
+│   ├── private.readme
+│   ├── production.in
+│   ├── quality.in
+│   ├── test.in
+│   └── travis.in
+├── setup.cfg
+├── setup.py
+└── {{cookiecutter.sub_dir_name}}
+    └── {{cookiecutter.library_name}}.py
+
+6 directories, 44 files
```

Comment:
What's this file for?

Reply:
Something I added to show dir structure, just missed removing it